app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
bad code style, fix it. Here is example: https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md#code-style